site/content/docs/5.3/components/list-group.md

```diff
@@ -101,42 +101,45 @@ Add `.list-group-flush` to remove some borders and rounded corners to render lis
 Add the `.list-group-numbered` modifier class (and optionally use an `<ol>` element) to opt into numbered list group items. Numbers are generated via CSS (as opposed to a `<ol>`s default browser styling) for better placement inside list group items and to allow for better customization.
 
 Numbers are generated by `counter-reset` on the `<ol>`, and then styled and placed with a `::before` pseudo-element on the `<li>` with `counter-increment` and `content`.
-
+<!-- Boosted mod: add d-flex on numbered list-group-item -->
 {{< example >}}
 <ol class="list-group list-group-numbered">
-  <li class="list-group-item">A list item</li>
-  <li class="list-group-item">A list item</li>
-  <li class="list-group-item">A list item</li>
+  <li class="list-group-item d-flex">A list item</li>
```

Comment:
Is it not possible to add a CSS rule instead targeting IDK `.list-group-numbered .list-group-item` so that the markup stays the same as Bootstrap?